technologies/varnish/monitor-varnish-cache/docker/Dockerfile

```
@@ -0,0 +1,30 @@
+## Designed to be a base layer to inherit from with the monitoring script and python 3 pre-installed.
+## The API_TOKEN and BASE_URL already injected in the build
+## --- TAKE CARE IN USING THIS APPROACH ---
+## The token will be visible on the image upon inspection
+## and therefore doesn't follow Docker's Best Practices
+## --- Internal/private Use Only ---
+ARG DISTRO_TAG=latest
+
+FROM alpine:${DISTRO_TAG} AS packages
+
+RUN apk update && apk upgrade \
+  && apk add python3 \
```

`py-pip` seems to be missing. Otherwise, I get `/bin/sh: pip3: not found`.
```suggestion
  && apk add python3 \
  && apk add py-pip \
```